### ISSUE TITLE
How to solve cross-domain issues？

### ISSUE THREAD

**Issue by wulisensen** [liyasthomas/postwoman#493]
Access to XMLHttpRequest at 'http://127.0.0.1:7001/stat/userSet' from origin 'https://postwoman.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by liyasthomas** [comment]
Possible duplicate of #433 #274 #128

**Comment by liyasthomas** [comment]
@shi-xuning install our browser extension (links are in README) to fix CORS issues.

**Comment by shi-xuning** [comment]
我已经尝试过安装浏览器扩展并且启用，但是好像并不起作用。 版本是0.17

**Comment by liyasthomas** [comment]
Please describe the steps to reproduce your current issue.

**Comment by shi-xuning** [comment]
目前我正在学习了解postwoman，我以下载zip包的方式在win10上安装并启动了它，通过访问http://localhost:3000，进行post请求报错，此时代理是打开状态且浏览器已安装了扩展
![image](https://user-images.githubusercontent.com/72735577/95958111-42b1c680-0e33-11eb-97a8-782e53f785b9.png)
![image](https://user-images.githubusercontent.com/72735577/95957983-0aaa8380-0e33-11eb-8404-d7bf6285ba07.png)
![image](https://user-images.githubusercontent.com/72735577/95958086-388fc800-0e33-11eb-8a55-9b71adc5e17b.png)
